packages/react-components/src/components/ControlBar.tsx

```diff
@@ -70,7 +70,14 @@ export const ControlBar = (props: ControlBarProps): JSX.Element => {
   }, [layout, styles?.root, theme]);
   return (
     <div className={mergeStyles(mainDivStyle)}>
-      <Stack className={controlBarClassName}>{props.children}</Stack>
+      <Stack
+        className={mergeStyles(controlBarClassName, {
+          borderRadius: (theme as PartialTheme)?.effects?.roundedCorner6,
```

Comment:
Can these be moved into the controlBarClass in the `.styles` file?

Reply:
Pulling things out of the theme outside the component is a bit complex. You need to use `useTheme`, but you can only do that from another `useX` function.

There is a supported way for this via registered names, but I don't think we're using that uniformly (yet).

So, soft suggestion here. Do it only if it's easy.

Reply:
We do it elsewhere with `getTheme`. But now I'm looking at it I'm not convinced this will update when the theme is updated:

https://github.com/Azure/communication-ui-library/blob/a205caeb26065412fff2e4c38b1aaeac76d6f921/packages/react-components/src/components/styles/VideoGallery.styles.ts#L14

![image](https://user-images.githubusercontent.com/2684369/134053686-9763a900-4467-4136-90a7-69ab97ebfdee.png)

Reply:
Yes, I can't move the merging of styles to a separate styles file because `getTheme` does not get the theme context from the `ThemeProvider`

Reply:
Oh good to know! I've created a bug to remove uses of getTheme: https://skype.visualstudio.com/SPOOL/_workitems/edit/2591054